data/pc/1.20.5/foods.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
is this supposed to be blank?

Reply:
Likely a failure of the extractor, this would need to be fixed. Doesn't seem like there was any changes to 1.20.5 food list so it may be possible to write a script to just update the item IDs.